admin/workspace-providers.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Workspace Providers enable a single Coder deployment to provision and manage
```

Comment:
Workspace providers aren't meant for customers yet. We should change this file to read:

> Workspace providers are slated for a future release. They will enable a single Coder deployment to provision .....

Or we should just remove this file.